Tasktive/Views/Sheets/TaskDetailsSheet.swift

```diff
@@ -30,21 +31,29 @@ struct TaskDetailsSheet: View {
         ) {
             VStack {
                 KFloatingTextField(text: $viewModel.title, title: TasktiveLocale.getText(.TITLE_INPUT_TITLE))
+
+                Button {
+                    onDelete()
+                } label: {
+                    (Text(Image(systemName: "trash")) + Text(" Delete"))
```

Comment:
Please add "Delete" in `Locales/en.js`, run `yarn localize` and write it as:

```swift
Text(localized: .LOCALE_KEY)
```

Reply:
I think it's better to use `.padding` to determine the space between the symbol and text.